src/feature/control/control_getinfo.c

```diff
@@ -325,6 +325,51 @@ getinfo_helper_current_time(control_connection_t *control_conn,
   return 0;
 }
 
+/**
+ * Switch between microdesc vs networkstatus descriptor dumps
```

Comment:
Please improve the function documentation here by making it more readable. Please see how the other GETINFO helper functions are documented and take a similar human readable approach.

Also please document the return value of the function.